pkg/revocation/db_test.go

```diff
@@ -151,3 +151,48 @@ func TestRevocationDB_Put_error(t *testing.T) {
 		}
 	})
 }
+
+func TestRevocationDB_List(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	testrevocation.RunDBs(t, func(t *testing.T, revDB extensions.RevocationDB, db storage.KeyValueStore) {
+		keys, chain, err := testpeertls.NewCertChain(2, storj.LatestIDVersion().Number)
+		require.NoError(t, err)
+		keys2, chain2, err := testpeertls.NewCertChain(2, storj.LatestIDVersion().Number)
+		require.NoError(t, err)
+
+		// test list no revocations, should not error
+		revs, err := revDB.List(ctx)
+		assert.NoError(t, err)
```

Comment:
`assert.NoError()` is almost never appropriate. If there is an error here, the things that happen afterward are not expected to work at all. Use `require.NoError()` here (and for all other `assert.NoError()` instances in this file) so that the test exits on error instead of trying to continue.

Reply:
good to know!